examples/hk.jl

```diff
@@ -142,22 +142,29 @@ agent_data, _ = run!(
 agent_data
 
 # Finally we run three scenarios, collect the data and plot it.
-using Plots
+using DataFrames, CairoMakie
+using Makie # hide
 using Random # hide
-gr() # hide
-
-plotsim(data, ϵ) = plot(
-    data.step,
-    data.new_opinion,
-    leg = false,
-    group = data.id,
-    title = "epsilon = $(ϵ)",
-)
 
-Random.seed!(42) # hide
```

Comment:
The seeds are useful so we don't have the documentation 'look' changing everytime we alter something in the repository. We need to keep it.